src/mocks/server.ts

```diff
@@ -46,7 +46,10 @@ export const mockErrorResponse = (
 
   server.use(
     http[method](`${baseUrl}${path}`, async () => {
-      return HttpResponse.text('lorem ipsum dolor sit amet', { status })
+      return HttpResponse.json(
```

Comment:
Denne har jeg tidligere endret fra .json til .text fordi feil fra kall til /api returnerte faktiske tekster og ikke json. Hensikten var å ha det så realistisk som mulig (oppdaget da jeg skrev tester for noe feilhåndtering)